numpyro/distributions/transforms.py

```diff
@@ -371,8 +371,8 @@ def log_abs_det_jacobian(self, x, y, intermediates=None):
 
 class MultivariateAffineTransform(LowerCholeskyAffine):
     def __init__(self, loc, scale_tril):
-        warnings.warn(DeprecationWarning,
-                      "MultivariateAffineTransform is renamed to LowerCholeskyAffine.")
+        warnings.warn("MultivariateAffineTransform is renamed to LowerCholeskyAffine.",
+                      DeprecationWarning)
```

Comment:
Sorry, I might have missed this earlier.`DeprecationgWarning` will be swallowed by application code. We should use `FutureWarning` instead. See https://docs.python.org/3/library/warnings.html#warning-categories.

Reply:
Thanks! This explains why I didn't see the warnings when I used `predictive.get_samples(...)`. :D